### PR TITLE
Angular Strict Template for app/exercises/file-upload/manage

### PR DESCRIPTION
**Pull request by SimonEntholzer**
### Motivation and Context

Some Angular strict template fixes.

### CODE REVIEW

**Review by ivohashamov — DISMISSED** [review]

File: src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html [hunk 1]
```diff
@@ -46,9 +46,9 @@ <h2 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
                     <div class="form-group-narrow flex-grow-1 ml-3">
                         <jhi-date-time-picker
                             labelName="{{ 'artemisApp.exercise.dueDate' | artemisTranslate }}"
-                            [(ngModel)]="fileUploadExercise.dueDate"
+                            [(ngModel)]="fileUploadExercise.dueDate!"
```
Comment (FrankeLukas) [inline src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html:49]: Why is this necessary here but not in line 40 of the same file which does basically the same? (Also, the date can be undefined, right?)

File: src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html [hunk 2]
```diff
@@ -57,16 +57,16 @@ <h2 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
...
                         (valueChange)="validateDate()"
-                        [(ngModel)]="fileUploadExercise.assessmentDueDate"
+                        [(ngModel)]="fileUploadExercise.assessmentDueDate!"
                     ></jhi-date-time-picker>
                     <span *ngIf="fileUploadExercise.assessmentDueDateError" class="invalid-feedback">{{ 'artemisApp.exercise.assessmentDueDateError' | artemisTranslate }}</span>
                 </div>
                 <div class="form-group">
                     <label class="form-control-label">{{ 'artemisApp.exercise.includedInOverallScoreLabel' | artemisTranslate }}</label>
-                    <div>
+                    <div *ngIf="fileUploadExercise.includedInOverallScore">
```
Comment (iozbeyli) [inline src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html:69]: Isn't this changing the behavior since now you do not show the component at all?

**Review by iozbeyli — COMMENTED** [review]
I added a couple of comments

File: src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html (hunk 1 above)
Comment (SimonEntholzer) [reply src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html]: you are right, in this place it was not necessary

File: src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html (hunk 2 above)
Comment (SimonEntholzer) [reply src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html]: I changed it back so it does not change the behaviour

**Review by krusche — APPROVED** [review]
